tests/integration/test_tableclasses_clusteredmetadatavectorcassandratable.py

```diff
@@ -264,3 +261,268 @@ def test_crud_partitioned_ann(self, db_session: Session, db_keyspace: str) -> No
+    #     for n_theta in range(N):
+    #         theta = n_theta * math.pi * 2 / N
+    #         group = "odd" if n_theta % 2 == 1 else "even"
+    #         t.put(
+    #             # row_id is tuple
+    #             row_id=(f"Z_theta_{n_theta}", n_theta + 1),
+    #             body_blob=f"Z_theta = {theta:.4f}",
+    #             vector=[math.cos(theta), math.sin(theta)],
+    #             partition_id=999,
+    #             metadata={
+    #                 group: True,
+    #                 "n_theta_mod_2": n_theta % 2,
+    #                 "group": group,
+    #             },
+    #         )
+
+    #     # retrieval
+    #     row_id = ("theta_1", 2)
+    #     theta_1 = t.get(row_id=row_id)
+    #     assert theta_1 is not None
+    #     assert abs(theta_1["vector"][0] - math.cos(math.pi * 2 / N)) < 3.0e-8
+    #     assert abs(theta_1["vector"][1] - math.sin(math.pi * 2 / N)) < 3.0e-8
+    #     assert theta_1["partition_id"] == 0
+    #     assert "row_id" in theta_1
+    #     assert isinstance(theta_1["row_id"], tuple)
+    #     assert theta_1["row_id"][0] == "theta_1"
+    #     assert theta_1["row_id"][1] == 2
+
+    #     # retrieval with metadata filtering
+    #     theta_1b = t.get(row_id=row_id, metadata={"odd": True})
+    #     assert theta_1b == theta_1
+    #     theta_1n = t.get(row_id=row_id, metadata={"even": True})
+    #     assert theta_1n is None
+
+    #     # ANN
+    #     # a vector halfway between 0 and 1 inserted above
+    #     query_theta = 1 * math.pi * 2 / (2 * N)
+    #     ref_vector = [math.cos(query_theta), math.sin(query_theta)]
+    #     ann_results1 = list(t.ann_search(ref_vector, n=4))
+    #     assert {r["row_id"] for r in ann_results1[:2]} == {
+    #         ("theta_1", 2),
+    #         ("theta_0", 1),
+    #     }
+    #     assert {r["row_id"] for r in ann_results1[2:4]} == {
+    #         ("theta_2", 3),
+    #         ("theta_15", 16),
+    #     }
+    #     # ANN with metadata filtering
+    #     ann_results_md1 = list(t.ann_search(ref_vector, n=4, metadata={"odd": True}))
+    #     assert {r["row_id"] for r in ann_results_md1[:2]} == {
+    #         ("theta_1", 2),
+    #         ("theta_15", 16),
+    #     }
+    #     assert {r["row_id"] for r in ann_results_md1[2:4]} == {
+    #         ("theta_3", 4),
+    #         ("theta_13", 14),
+    #     }
+
+    #     # retrieval on 999
+    #     row_id = ("Z_theta_1", 2)
+    #     ztheta_1 = t.get(row_id=row_id, partition_id=999)
+    #     assert ztheta_1 is not None
+    #     assert abs(ztheta_1["vector"][0] - math.cos(math.pi * 2 / N)) < 3.0e-8
+    #     assert abs(ztheta_1["vector"][1] - math.sin(math.pi * 2 / N)) < 3.0e-8
+    #     assert ztheta_1["partition_id"] == 999
+    #     assert "row_id" in ztheta_1
+    #     assert isinstance(ztheta_1["row_id"], tuple)
+    #     assert ztheta_1["row_id"][0] == "Z_theta_1"
+    #     assert ztheta_1["row_id"][1] == 2
+
+    #     t.clear()
```

Comment:
is this the test I added that is broken?

Reply:
i think it is... lets just remove these comments in favor of your colbert-style test below.

Reply:
That's correct. Shall I remove it entirely? (no big loss in view of the test I added below)

Reply:
yes pls remove entirely